lib/dotenv_validator.rb

```diff
@@ -31,13 +31,13 @@ def self.analyze_variables
 
       valid =
         case Regexp.last_match(1)
-        when "int", "integer" then integer?(value)
-        when "float" then float?(value)
-        when "str", "string" then true
-        when "email" then email?(value)
-        when "url" then url?(value)
-        when "bool", "boolean" then boolean?(value)
-        when "uuid" then uuid?(value)
+        when *integer_options then integer?(value)
+        when *float_options then float?(value)
+        when *string_options then string?(value)
+        when *email_options then email?(value)
+        when *url_options then url?(value)
+        when *boolean_options then boolean?(value)
+        when *uuid_options then uuid?(value)
```

Comment:
is this extraction to methods that return an array really needed? I feel this adds an extra layer that doesn't really add much (you not have to scroll to see what are the valid options and it's not re-used anywhere)

Reply:
I did it because I thought there were many options in each `when` statement. do you suggest returning it how it was?